### ISSUE TITLE
Run scripts periodically by configurable timers

### ISSUE THREAD

**Issue by djkrose** [djkrose/7DTD-ScriptingMod#7]
It should be possible to run scripts repeatedly based on timers, similar to a cronjob.

**Comment by ddumont** [comment]
Is setimout or setinterval not available in the js runtime?

**Comment by djkrose** [comment]
No, because this is part of the window object, which does not exist without browser. I might add these two methods later, but it would still require someone to execute a command initially, after every server restart.

It is also problematic to have long-running scripts, because commands are executed on the main thread and freeze the server while executing. Adding the possibility to delay execution with settimeout is not trivial.
